.circleci/config.yml

```diff
@@ -343,6 +362,9 @@ workflows:
           filters:
             branches:
               only: master
+      - solidity-checking:
+          requires:
+            - build-docker
```

Comment:
Doesn't require the `build-docker` job (which build the custom rust image), so just listing the job name without a `requires` section should be enough.